rfqmsg/buy_request.go

```diff
@@ -99,22 +101,25 @@ func NewBuyRequestMsgFromWire(wireMsg WireMessage,
 			"request")
 	}
 
-	// Extract the suggested rate tick if provided.
-	var bidPrice lnwire.MilliSatoshi
+	// Extract the suggested asset to BTC rate if provided.
+	//
+	// TODO(ffranr): Temp solution.
+	var suggestedAssetRate *BigIntFixedPoint
 	msgData.SuggestedRateTick.WhenSome(
 		func(rate tlv.RecordT[tlv.TlvType4, uint64]) {
-			bidPrice = lnwire.MilliSatoshi(rate.Val)
+			r := NewBigIntFixedPoint(rate.Val, 0)
```

Comment:
Assumes scale.

Reply:
part of temp solution

It will be fully threaded through in follow-up PR as noted in the [PR desc](https://github.com/lightninglabs/taproot-assets/pull/1136#issue-2546061589)

Reply:
Yes. Everything marked with `// TODO(ffranr): Temp solution.` in the latest commits for this PR.